spec/02-integration/05-proxy/10-balancer/01-healthchecks_spec.lua

```diff
@@ -1056,17 +1089,23 @@ for _, strategy in helpers.each_strategy() do
               server1:start()
               server2:start()
 
-              oks, fails = bu.client_requests(bu.SLOTS * 2, api_host)
-              assert.same(bu.SLOTS * 2, oks)
-              assert.same(0, fails)
+              -- XXX: flaky
```

Comment:
It works well in the local environment but always fails in ci test. Can someone tell me why?

Reply:
https://github.com/Kong/kong/pull/8715/files#r863841171

Reply:
Thanks for your kindly help, Let's discuss this issue there.